app/helpers/views_helper.rb

```diff
@@ -0,0 +1,9 @@
+module ViewsHelper
+  def get_sar_heading(kase)
+    if @case.current_state == "rejected" # rubocop:disable Rails/HelperInstanceVariable
```

Comment:
You are passing `@case` into the method as `kase` so you can use that variable. That will take care of the warning.

Reply:
You should probably check that the case type is an Offender SAR as well, just in case any of the other case types ever have a "rejected" state in the future.